src/utils/ProfileUtils.ts

```diff
@@ -0,0 +1,29 @@
+import ZipProvider from "../providers/generic/zip/ZipProvider";
```

Comment:
The imports in this project generally is a mess as there's no linting to autosort them. Regardless I personally like to at least try to get them right when creating new files. By "right" I mean "what I'm used to", which is the Python way, i.e. "standard library imports" (e.g. path) separated by a blank line from 3rd party imports (e.g. yaml) separated by a blank line from local imports. Each section should be sorted alphabetically based on the import path, e.g. the part after "from".

That's my personal preference, so this won't be a blocker now or later. If you choose, this can be fixed in the next PR.